src/FTPC.jl

```diff
@@ -502,24 +473,15 @@ Establish connection to FTP server.
 returns ctxt::ConnContext
 """ ->
 function ftp_connect(options::RequestOptions=RequestOptions())
-    if options.blocking
-        ctxt = false
-        try
-            ctxt = setup_easy_handle(options)
-
-            # ping the server
-            resp = ftp_command(ctxt, "LIST")
-
-            ctxt.options.blocking = ctxt.options.reset_blocking
-
-            return ctxt, resp
+    ctxt = setup_easy_handle(options)
+    try
+        # ping the server
+        resp = ftp_command(ctxt, "LIST")
 
-        catch
-            cleanup_easy_context(ctxt)
-            rethrow()
-        end
-    else
-        return remotecall(ftp_connect, myid(), set_opt_blocking(options))
+        return ctxt, resp
+    catch
+        cleanup_easy_context(ctxt)
+        rethrow()
```

Comment:
`ctxt` is how we maintain a persistent connection. Users use `ftp_connect` to make the connection and get `ctxt`.